kitchen/modules/hello-substrate/src/lib.rs

```diff
@@ -134,4 +116,6 @@ mod tests {
 			assert_eq!(HelloSubstrate::last_value(), 11u64);
 		});
 	}
+
+	// TODO: find out how to construct signed extrinsics in runtime testing
```

Comment:
You mean like actually cryptographically signed? What's the motivation for doing that? The technique you've used in the previous few tests is pretty standard.

Reply:
Definitely, I think this idea was/is overkill. I'll delete this line at the next chance.